### ISSUE TITLE
Run a mini-migrator to rename build->host before osx-arm64 migrators

### ISSUE THREAD

**Issue by isuruf** [regro/cf-scripts#1268]
For eg: https://github.com/regro-cf-autotick-bot/jupyter_core-feedstock/blob/0088e71f7e8a7b7998bef034d3059239a1b1926a/recipe/meta.yaml#L20

**Comment by isuruf** [comment]
Note that this doesn't fix the CrossPythonMigrator at https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators/cross_compile.py#L147 because it is looking at the requirements from the recipe from master.

**Comment by beckermr** [comment]
I'm not sure what you mean @isuruf.

**Comment by isuruf** [comment]
CrossPythonMigrator works only if `python` is in `host`. build 2 host mini migrator changes python from `build` to `host`, but when `CrossPythonMigrator` checks for `python` it is still in build because the metadata in the graph hasn't changed.

**Comment by CJ-Wright** [comment]
right, filter and migrate need to call the actual feedstock loader

**Comment by beckermr** [comment]
OK. So we should force the graph node to be remade when doing this one. Is that going to break other things or is it ok?

**Comment by CJ-Wright** [comment]
I think this will be ok, so long as we don't store that data anywhere (eg. in the graph)
